### ISSUE TITLE
Media messages have stale LDAT, pointing to previous memes server

### ISSUE THREAD

**Issue by Entryist** [getZION/app#83]
A lot of messages have media_tokens that when decoded via `parseLDAT` have a `uri` of `memes.n2n2.chat`, the address of our previous memes server.

Those messages display as empty content, and may cause issues elsewhere in the app. Immediately after experiencing this issue, the previous "empty messages" bug occurred, where message content throughout the app fails due to decryption error.

I attempted a workaround by swapping out every instance of `memes.n2n2.chat` with `memes.getzion.com`, but it didn't seem to help. From `/src/views/chat/msg/hooks.ts`:

```
const theHost = ldat.host === 'memes.n2n2.chat' ? 'memes.getzion.com' : ldat.host
const url = `https://${theHost}/file/${media_token}`
```

**Comment by Entryist** [comment]
The "MEMETICS" community has a number of these messages with stale data.

**Comment by masterial** [comment]
Yes, it won't work. The context key got reset when the domain name changed.

**Comment by Entryist** [comment]
Fortunately these are all quite old messages. For now I can just not query relay for messages this old and older.

**Comment by Entryist** [comment]
Setting the earliest possible date for message retrieval to Aug 15, 2021 - four days after the app code was changed to the new realm server - and all messages in the example community where I first saw this (MEMETICS) load properly.

So far this seems LDAT issue MIGHT be the root cause of our earlier unsolved ['empty messages' issue](https://github.com/getZION/internal/issues/1). Will keep investigating.

![memetics](https://user-images.githubusercontent.com/14167547/146579926-38f791eb-d149-4e92-bc43-d2071dc33257.png)

**Comment by Entryist** [comment]
Something is still calling the `tribes.n2n2.chat` and `memes.n2n2.chat` endpoints.

![stillcalling](https://user-images.githubusercontent.com/14167547/146595905-bead3415-6079-4a2f-be2d-44171bcbf57f.png)

Currently, all messages after yesterday afternoon are showing as "empty messages" where decryption is failing with this error generated in the Xcode console:

`Error Domain=NSOSStatusErrorDomain Code=-50 "RSAdecrypt wrong input (err -27)" UserInfo={numberOfErrorsDeep=0, NSDescription=RSAdecrypt wrong input (err -27)}`

Copy-pasting the device privkey and encoded message content into web-based RSA decryption sites shows messages before the yesterdayish cutoff are decrypting fine, but after that are generating an error.

Not sure exactly how these two issues are related, but they seem to be popping up together and probably are.

**Comment by Entryist** [comment]
One URL it's failing to reach is: `https://memes.n2n2.chat/public/92N2A86OaoocWD6t7cv40wKG1HYF8rGs8nYnqKrYcDE=`

Replacing the base URL with memes.getzion.com, we get `https://memes.getzion.com/public/92N2A86OaoocWD6t7cv40wKG1HYF8rGs8nYnqKrYcDE=`

... which is this image:
![Image (2)](https://user-images.githubusercontent.com/14167547/146597013-10eeec4c-4d28-4fd8-be54-ea7f2d4cbb5e.jpg)

...presumably the profile picture of Zion support user or a community?

**Comment by Entryist** [comment]
A second URL it's failing to reach is: `https://tribes.n2n2.chat/tribes/YNzR0h_54n6O1zjeKKEM6vSzwO6yq3N-3aYSpAaeZ04C72dYISzw5UQEAeB3nvwVXspEnAFavDaE0o-bmxZiw_2CNpL1`

Replacing the base URL with the new communities server:
https://communities.getzion.com/tribes/YNzR0h_54n6O1zjeKKEM6vSzwO6yq3N-3aYSpAaeZ04C72dYISzw5UQEAeB3nvwVXspEnAFavDaE0o-bmxZiw_2CNpL1

...gives this:

`
{"uuid":"YNzR0h_54n6O1zjeKKEM6vSzwO6yq3N-3aYSpAaeZ04C72dYISzw5UQEAeB3nvwVXspEnAFavDaE0o-bmxZiw_2CNpL1","owner_pubkey":"03fc010914cd0b9950f128f1e91c83da9d718b688aa86c5ed5b8bbf521a2f41fc1","owner_alias":"Zion Support","group_key":"MIIBCgKCAQEAmtOQIDjRoYosABbgpH3CGx7dhxA/n9uW5k9yTrw85fJGhhcjXRC+fRKWjDo0ORf0O00TN4ot/EqMl5L0YcKBFpAOsvpohWRGdDxcVQ4sCF6GdbQJqfIzaNwTdcrq29kGKbLpcDJ7tHgSkFuXJcjhcEd/Ui5DAOjg6THumDtXTJA0V2Tk4b+CJiS68J8MgCP5hCTKdgZx665ttMfeP0Zcm+FYpYFOheXfjE7siaXdkEd2S+4jGACfPNfANEeFi2pSUMzAFu5iVcmolR/F6g25WTHixhMy27TGF+SzbriiXyVfGpyleUFNG+J66mRwQDStLFaOfl7ce7LiegXEi2JrFwIDAQAB","name":"Zion","description":"Welcome to Zion Tech Support! This chat is for general Q\u0026A, support and feedback.   For specific technical queries, please email our Engineering Team at support@getzion.zendesk.com.   If you’re new here and you’re not sure how to get started, please read our Zion Guide:  https://guide.getzion.com/  For cancellations, billing and refunds, please visit:  htttps://shop.getzion.com/account  As with any new tech platforms, we’re constantly updating, troubleshooting and improving every single day — Thank you for being on this exciting journey with us!","tags":["support","zion"],"img":"https://memes.getzion.com/public/92N2A86OaoocWD6t7cv40wKG1HYF8rGs8nYnqKrYcDE=","price_to_join":0,"price_per_message":0,"escrow_amount":10,"escrow_millis":0,"created":"2021-06-30T20:19:31.134473Z","updated":"2021-12-17T18:46:15.220944Z","member_count":1237,"unlisted":false,"private":false,"deleted":false,"app_url":"","feed_url":"","last_active":1639766775,"bots":"[]","owner_route_hint":""}
`
